lunatrace/bsl/ingest-worker/pkg/epss/fetch.go

```diff
@@ -0,0 +1,120 @@
+// Copyright by LunaSec (owned by Refinery Labs, Inc)
+//
+// Licensed under the Business Source License v1.1
+// (the "License"); you may not use this file except in compliance with the
+// License. You may obtain a copy of the License at
+//
+// https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package epss
+
+import (
+  "bytes"
+  "compress/gzip"
+  "encoding/csv"
+  "fmt"
+  "github.com/rs/zerolog/log"
+  "io"
+  "net/http"
+  "strconv"
+  "time"
+)
+
+// EpssVulnerability has a CVE, Epss, and Percentile
+type EpssVulnerability struct {
+  Cve        string
+  Epss       float64
+  Percentile float64
+}
+
+func FetchEpssScores() (epssScores []EpssVulnerability, err error) {
```

Comment:
nit: I would move these into the function and only allocate the epssScores array after we've read the CSV. We can save allocations by [specifying the capacity of the slice](https://github.com/uber-go/guide/blob/master/style.md#prefer-specifying-container-capacity) when we create it.